src/System.Drawing.Common/src/System/Drawing/Graphics.Unix.cs

```diff
@@ -197,19 +197,10 @@ public void CopyFromScreen(int sourceX, int sourceY, int destinationX, int desti
             {
                 CopyFromScreenX11(sourceX, sourceY, destinationX, destinationY, blockRegionSize, copyPixelOperation);
             }
-            else if (SafeNativeMethods.Gdip.UseCarbonDrawable)
+            else
             {
-                CopyFromScreenMac(sourceX, sourceY, destinationX, destinationY, blockRegionSize, copyPixelOperation);
+                throw new NotImplementedException();
```

Comment:
Maybe a Debug.Fail, too?

Reply:
You'll hit this when `UseX11Drawable` is `false`, which is the default on macOS. A `PlatformNotSupportedException` would be better, I'll update that.